torch/autograd/_functions/linalg.py

```diff
@@ -102,3 +102,42 @@ def backward(ctx, grad_output, grad_LU=None):
         grad_b, _ = torch.gesv(grad_output, a.t())
         grad_a = -torch.mm(grad_b, X.t())
         return grad_b, grad_a
+
+class Symeig(Function):
+
+    @staticmethod
+    def forward(ctx, input, eigenvectors=False, upper=True):
+        ctx.eigenvectors = eigenvectors
```

Comment:
Do you use it elsewhere? Also, what happens if `eigenvectors=False`? The backward requires then, right?